windows/9.3.0/sitecore-assets/tools/entrypoints/sitecore-xc-engine/Production.ps1

```diff
@@ -0,0 +1,122 @@
+# setup
+$ErrorActionPreference = "STOP"
+
+Import-Module WebAdministration
+
+function Wait-WebItemState
+{
+    param(
+        [ValidateNotNullOrEmpty()]
+        [string]$IISPath
+        ,
+        [ValidateSet("Started", "Stopped")]
+        [string]$State
+    )
+
+    while ($true)
+    {
+        Write-Host "### Waiting on item '$IISPath' state to be '$State'..."
+
+        try
+        {
+            $item = Get-Item -Path $IISPath
+
+            if ($null -ne $item -and $item.State -ne $State)
+            {
+                if ($State -eq "Started")
+                {
+                    $item = Start-WebItem -PSPath $IISPath -Passthru -ErrorAction "SilentlyContinue"
+                }
+                elseif ($State -eq "Stopped")
+                {
+                    $item = Stop-WebItem -PSPath $IISPath -Passthru -ErrorAction "SilentlyContinue"
+                }
+            }
+        }
+        catch
+        {
+            $item = $null
+        }
+
+        if ($null -ne $item -and $item.State -eq $State)
+        {
+            Write-Host "### Waiting on item '$IISPath' completed."
+
+            break
+        }
+
+        Start-Sleep -Milliseconds 500
+    }
+}
+
+# print start message
+Write-Host ("### Commerce Engine Production ENTRYPOINT, starting...")
+
+# wait for w3wp to stop
+while ($true)
+{
+    $processName = "w3wp"
+
+    Write-Host "### Waiting for process '$processName' to stop..."
+
+    $running = [array](Get-Process -Name $processName -ErrorAction "SilentlyContinue").Length -gt 0
+
+    if ($running)
+    {
+        Stop-Process -Name $processName -Force -ErrorAction "SilentlyContinue"
+    }
+    else
+    {
+        Write-Host "### Process '$processName' stopped..."
+
+        break;
+    }
+
+    Start-Sleep -Milliseconds 500
+}
+
+# wait for application pool to stop
+Wait-WebItemState -IISPath "IIS:\AppPools\DefaultAppPool" -State "Stopped"
+
+# Set connection string details
+c:/tools/scripts/Set-CommerceEngineConnectionString.ps1 -userName $env:SQL_USER -password $env:SQL_PASSWORD -server $env:SQL_HOST -globalDatabaseName $env:SQL_COMMERCE_GLOBAL_DB_NAME -sharedEnvironmentDatabaseName $env:SQL_COMMERCE_SHAREDENVIRONMENT_DB_NAME
```

Comment:
please remove this line as it's pointing to a non-existing powershell script. There's an open PR for this to fix the 9.2 XC entrypoint script https://github.com/Sitecore/docker-images/pull/223

Reply:
@joostmeijles can you verify #233, and add into #230

Reply:
Verified and now included fix from #223